frontend/src/lib/components/ChartFilter/chartFilterLogic.ts

```diff
@@ -2,43 +2,50 @@ import { kea } from 'kea'
 import { router } from 'kea-router'
 import { objectsEqual } from 'lib/utils'
 import { chartFilterLogicType } from './chartFilterLogicType'
-import { ChartDisplayType, ViewType } from '~/types'
+import { ChartDisplayType, FunnelVizType, ViewType } from '~/types'
+
+function isFunnelVizType(filter: FunnelVizType | ChartDisplayType): filter is FunnelVizType {
```

Comment:
Again, previous comments about why we need `ViewType` and `FunnelVizType` and `ChartDisplayType` apply here.